blog/modules/ROOT/pages/from-gitbook-to-antora.adoc

```diff
@@ -101,6 +101,12 @@ We also explicitly define the start page.
 By default, Antora looks for a file named `index.adoc` in the `ROOT` module.
 So if your start page is already named `index.adoc`, you can remove the `start_page` key.
 
+[NOTE]
+====
+The name of the component is determined from the `name` key in the component descriptor.
+We happened to name the directory the component descriptor is in `book` also, but this does not affect the name of the component.
```

Comment:
I don't understand this sentence 🤔
I'm not a native speaker so it might be valid but hard to read in my opinion:

> to name the directory the component descriptor is in `book` also

Reply:
See if my change makes it any clearer.... I agree it's unclear.